### ISSUE TITLE
Project is not set up automatically 

### ISSUE THREAD

**Issue by hans-cyfrin** [Cyfrin/audit-repo-cloner#14]
The cloner script fails to set up the project with the error message below.
```bash
Error occurred while setting up project board: 410 {"message": "Projects are disabled for this repository", "documentation_url": "https://docs.github.com/v3/projects"}
Please set up project board manually.
```
I don't know if this is inevitable if we are going to use the board template.

**Comment by giovannidisiena** [comment]
Yeah, it seems the GitHub API does not work correctly at the moment for setting up project boards. I looked into it before but couldn't figure out a solution, so we may have to accept this for now. It's not too involved to set up manually from the template, at least.

**Comment by giovannidisiena** [comment]
@hans-cyfrin this clears it up https://stackoverflow.com/a/73760739.

tl;dr: it isn't possible with PyGithub but maybe using the GraphQL API...

**Comment by hans-cyfrin** [comment]
Ah, I see. I need to familiarize myself with GraphQL.

**Comment by giovannidisiena** [comment]
Update: I managed to create a ProjectV2 using the GraphQL API at this commit 904a9c8 but we are still somewhat limited by the inability to modify the status field (for column headings). See below:
- https://github.com/orgs/community/discussions/44265
- https://github.com/orgs/community/discussions/38225#discussioncomment-5956771

One workaround, as described in the discussion, is to use a different field as the column heading but I think for now it may be easiest and best to simply click one button in the UI to create a new project from the template.
